jupyter-docker/jupyter_install_server_extension.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
```

Comment:
I wonder if this is a good opportunity to organize the `jupyter-docker` folder a bit, perhaps by creating a `scripts` folder with subfolders like `extension-installation`, `pip-installation`, etc.